safe/definitions/fields.py

```diff
@@ -2234,6 +2238,7 @@
     'key': 'affected_productivity_cost_field',
     'name': tr('Affected Productivity Cost'),
     'field_name': 'affected_productivity_cost',
+    'header_name': tr('Production Cost'),
```

Comment:
I think it's my fault. I will make a PR to make sure they are uniform. And the diagram that I said this morning.